lib/logstash/inputs/http.rb

```diff
@@ -142,10 +143,12 @@ def run(queue)
           event.set("headers", req)
           decorate(event)
           queue << event
+          metric.increment(:events)
```

Comment:
purpose it to track events_in for the inputs, for such input we could for example have 10 events for 1 http request (as example). are you imagining something namespaced for the input? or something general for all the inputs?